etc/skydive.yml.default

```diff
@@ -175,11 +175,12 @@ logging:
     - stderr
   # - file
   # - syslog
+  # - json
   # configuration of the 'file' backend
   # file:
   #   path: /var/log/skydive.log
-  # format of the logging messages
-  # format: "%{color}%{time} %{id} %{shortfile} %{shortpkg} %{longfunc} > %{level:.4s} %{id:03x}%{color:reset} %{message}"
+  # configuration encoder could be for all backends or for specific one
+  #   encoder: json
```

Comment:
not correctly indented, you added it as sub element of file,should sub element of logging